denote.el

```diff
@@ -658,25 +644,22 @@ Like `denote--completion-table' but also disable sorting."
   (when (not (file-directory-p denote-directory))
     (make-directory denote-directory :parents)))
 
-(defvar denote-user-enforced-denote-directory nil
```

Comment:
Would it be worth marking denote-user-enforced-denote-directory with define-obsolete-variable-alias so people who are using it in their own code have a chance to update it before it's dropped completely?

Reply:
Done!